src/org/labkey/test/tests/ClientAPITest.java

```diff
@@ -101,34 +106,25 @@
     private final static File TEST_XLS_DATA_FILE = TestFileUtils.getSampleData("dataLoading/excel/ClientAPITestList.xls");
     private final static String SUBFOLDER_LIST = "subfolderList"; // for cross-folder query test
     private static final String OTHER_PROJECT_LIST = "otherProjectList"; // for cross-project query test
-    public final static ListHelper.ListColumnType LIST_KEY_TYPE = ListHelper.ListColumnType.AutoInteger;
-    public final static String LIST_KEY_NAME = "Key";
     // Add tricky characters to assay name to check for regressions
-    // 36077: SelectRows: SchemaKey decoding of public schema name causes request failure
+    // Issue 36077: SelectRows: SchemaKey decoding of public schema name causes request failure
     protected static final String TEST_ASSAY = "TestAssay1" + TRICKY_CHARACTERS;
     protected static final String TEST_ASSAY_DESC = "Description for assay 1";
-    public final static ListHelper.ListColumn[] LIST_COLUMNS = new ListHelper.ListColumn[]
-    {
-        new ListHelper.ListColumn("FirstName", "First Name", ListHelper.ListColumnType.String, "The first name"),
-        new ListHelper.ListColumn("LastName", "Last Name", ListHelper.ListColumnType.String, "The last name"),
-        new ListHelper.ListColumn("Age", "Age", ListHelper.ListColumnType.Integer, "The age")
-    };
-    static
-    {
-        LIST_COLUMNS[0].setRequired(true);
-        LIST_COLUMNS[1].setRequired(true);
-    }
-
-    public final static String[][] TEST_DATA =
-    {
-        { "1", "Bill", "Billson", "34" },
-        { "2", "Jane", "Janeson", "42" },
-        { "3", "John", "Johnson", "17" },
-        { "4", "Mandy", "Mandy;son", "32" },
-        { "5", "Norbert", "Norbertson", "28" },
-        { "6", "Penny", "Pennyson", "38" },
-        { "7", "Yak", "Yakson", "88" },
-    };
+    private static final List<FieldDefinition> LIST_COLUMNS = List.of(
+        new FieldDefinition("FirstName", FieldDefinition.ColumnType.String).setLabel("First Name").setDescription("The first name").setRequired(true),
+        new FieldDefinition("LastName", FieldDefinition.ColumnType.String).setLabel("Last Name").setDescription("The last name").setRequired(true),
+        new FieldDefinition("Age", FieldDefinition.ColumnType.Integer).setLabel("Age").setDescription("The age")
+    );
+
+    public final static String TEST_DATA =
+        "Key\tFirstName\tLastName\tAge\n" +
+        "1\tBill\tBillson\t34\n" +
+        "2\tJane\tJaneson\t42\n" +
+        "3\tJohn\tJohnson\t17\n" +
+        "4\tMandy\tMandy;son\t32\n" +
+        "5\tNorbert\tNorbertson\t28\n" +
+        "6\tPenny\tPennyson\t38\n" +
+        "7\tYak\tYakson\t88\n";
```

Comment:
Since this is only used in one place, why leave it as public? Why not move it to where it is used and create and add to the list using Map.of(...)?